docs/format/index.md

```diff
@@ -175,7 +175,7 @@ Postings sections store monotonically increasing lists of series references that
 └─────────────────────────────────────────┘
 ```
 
-The sequence of postings sections is finalized by an [offset table](#offset-table) pointing to the beginning of each postings section for a given set of label names.
+The sequence of postings sections is finalized by an [offset table](#posting-offset-entry) pointing to the beginning of each postings section for a given set of label names.
```

Comment:
This is quite confusing to me. The section is finalized by an Offset table itself NOT the posting / label one. So the full sentence should look like:
```suggestion
The sequence of postings sections is finalized by an [offset table](#offset-table) containing [posting offset entries](#posting-offset-entry) that points to the beginning of each postings section for a given set of label names.
```
and same for labels, right?

Reply:
yep that will do as well.